docs/source/features/parallelisms.rst

```diff
@@ -212,33 +275,6 @@ Visit our source code for more insights into the implementation:
 - `Transformer Engine attention modules <https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/pytorch/attention.py>`_
 
 
-Expert Parallelism
-^^^^^^^^^^^^^^^^^^
-Expert Parallelism (EP) is a type of model parallelism that distributes experts of an MoE across GPUs.
-
-.. image:: ../nlp/nemo_megatron/images/ep.png
-    :align: center
-    :width: 800px
-    :alt: Expert Parallelism
-
-Enable Expert Parallelism
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To enable EP, set ``model.expert_model_parallel_size`` to the desired expert parallel size. For example, if the model has six experts (``model.num_moe_experts=6``), then setting ``model.expert_model_parallel_size=3`` results in each GPU processing two experts. The number of experts should be divisible by the expert parallel size.
-
-   .. code-block:: yaml
-
-       expert_model_parallel_size: 3  # Set EP to 3
-
-For further information on configuration, refer to the following documentation: `NeMo Megatron GPT Config <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/conf/megatron_gpt_config.yaml#L68>`_.
-
-
-Implement Expert Parallelism
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The NeMo implementation of Expert Parallelism uses functionality from Megatron Core. Please consult the `Megatron Core MoE layer <https://github.com/NVIDIA/Megatron-LM/blob/e2ec14ab5690fead7e33760b0f8fb20c83b4fd1f/megatron/core/transformer/moe/moe_layer.py#L29>`_ for more MoE implementation details.
-
-
 Parallelism nomenclature
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
EEdits to Line 281

The following figure provides an illustration of terms that you might encounter in the NeMo Megatron codebase.